streamingpro-mlsql/src/main/java/streaming/dsl/mmlib/algs/SQLDiscretizer.scala

```diff
@@ -220,10 +229,14 @@ class SQLDiscretizer(override val uid: String) extends SQLAlg with Functions wit
       |
       |train data1 as Discretizer.`/tmp/model`
       |where method="bucketizer"
-      |and `fitParam.0.inputCol`="a"
-      |and `fitParam.0.splitArray`="-inf,0.0,1.0,inf"
-      |and `fitParam.1.inputCol`="b"
-      |and `fitParam.1.splitArray`="-inf,0.0,1.0,inf";
+      |and `inputCol`="a"
+      |and `splits`="-inf,0.0,1.0,inf";
+      |
+      |train data1 as Discretizer.`/tmp/discretizer`
+      |where method="quantile"
+      |and `inputCol`="a"
+      |and `outputCol`="a_v"
+      |and `numBuckets`="3";
       |;
     """.stripMargin)
```

Comment:
Good job! Should we leave some explanation in codeExample, such as the meaning of `splits`.

In addition, I have not seen an example of how to register and how to use it. Can you add it?